crates/autopilot/src/run_loop.rs

```diff
@@ -248,6 +248,12 @@ impl RunLoop {
                                     buy_amount: order.buy_amount,
                                 })
                                 .collect(),
+                            clearing_prices: participant
+                                .solution
+                                .clearing_prices
+                                .clone()
+                                .into_iter()
```

Comment:
nit: To avoid cloning the whole structure only to drain it immediately afterwards.
```suggestion
                                .iter()
                                .cloned()
```